lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore

```diff
@@ -0,0 +1,49 @@
+<div class="cohort-discussions-nav">
+    <form action="" method="post" id="cohort-coursewide-discussions-form" class="cohort-coursewide-discussions-form">
+        <h3 class="subsection-title"><%- gettext('Designate discussion topics as divided by cohort') %></h3>
+        <div class="wrapper cohort-management-supplemental">
+            <div class="form-fields">
+                <div class="form-field">
+                    <div class="coursewide-discussion-topics">
+                        <h3 class="subsection-title"><%- gettext('Coursewide Discussion Topics') %></h3>
+                        <p><%- gettext('You can optionally designate coursewide discussions topics to be cohorted.') %></p>
+                        <div class="field">
+                            <ul class="discussions-wrapper"><%= coursewideTopics %></ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="form-actions">
+                <button class="form-submit button action-primary action-save"><%- gettext('Save') %></button>
+            </div>
+        </div>
+    </form>
+
+    <hr class="divider divider-lv1" />
+
+    <form action="" method="post" id="cohort-inline-discussions-form" class="cohort-inline-discussions-form">
+        <div class="wrapper cohort-management-supplemental">
+            <div class="form-fields">
+                <div class="form-field">
+                    <div class="inline-discussion-topics">
+                        <h3 class="subsection-title"><%- gettext('Content Specific Discussion Topics') %></h3>
+                        <p><%- gettext('All other content-specific discussion topics, defined through inline discussion components, can also be designated as cohorted. You can cohort all discussion topics, or control specific areas below.') %></p>
```

Comment:
Same comment as above, about making this text more active guidance.
Suggested text: "Select the content-specific discussion topics that you want to divide by cohort. To specify that all existing and future content-specific discussion topics are divided by cohort, select **Always Cohort Content-Specific Discussion Topics**.
@explorerleslie @marcotuts @muzaffaryousaf